hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneAtRestEncryption.java

```diff
@@ -79,27 +78,23 @@
 import static org.apache.hadoop.hdds.HddsConfigKeys.OZONE_METADATA_DIRS;
 import static org.apache.hadoop.hdds.client.ReplicationFactor.ONE;
 import static org.apache.hadoop.hdds.client.ReplicationType.RATIS;
-
-import org.junit.AfterClass;
-import org.junit.Assert;
-import org.junit.BeforeClass;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.apache.ozone.test.tag.Flaky;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
 import org.mockito.Mockito;
 
-/**
- * This class is to test all the public facing APIs of Ozone Client.
- */
-@RunWith(Parameterized.class)
-public class TestOzoneAtRestEncryption {
-
-  @Parameterized.Parameters
-  public static Collection<BucketLayout> data() {
-    return Arrays.asList(
-        BucketLayout.FILE_SYSTEM_OPTIMIZED,
-        BucketLayout.OBJECT_STORE);
-  }
+class TestOzoneAtRestEncryption {
```

Comment:
Is the class javadoc removed here intentional?

Reply:
Yes, it was leftover from `TestOzoneRpcClient`.

```
hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestSecureOzoneRpcClient.java
84: * This class is to test all the public facing APIs of Ozone Client.

hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClient.java
32: * This class is to test all the public facing APIs of Ozone Client.
```